lib/lightning_web/live/project_live/index.html.heex

```diff
@@ -2,14 +2,14 @@
   <:header>
     <LayoutComponents.header current_user={@current_user}>
       <:title><%= @page_title %></:title>
-      <%= live_patch to: Routes.project_index_path(@socket, :new) do %>
+      <.link patch={Routes.project_index_path(@socket, :new)}>
```

Comment:
Could we use the sigil ~p here for compiled routes? I think for this one it would be `~p"/settings/projects/new"`

Reply:
Done